src/remote/playwrightServer.ts

```diff
@@ -33,14 +33,15 @@ export interface PlaywrightServerDelegate {
 
 export type PlaywrightServerOptions = {
   acceptForwardedPorts?: boolean
+  onDisconnect?: () => void;
```

Comment:
I don't see any usages, drop it for now?

Reply:
I use it for a demo, see later in the standup. Since it's only an internal API, it should be fine?

Reply:
If it's just for the demo you can use a local change? We normally stick to the rule of not adding unused code but if you really need it to be in the repo for the demo it's okay.

Reply:
(not exposed API, thats why I landed it for now. If we publish it at some point we re-consider the whole use of the API)